src/components/text-editor/examples/text-editor-prefilled-value.tsx

```diff
@@ -0,0 +1,54 @@
+import { LimelTextEditorCustomEvent } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+/**
+ * With prefilled value
+ */
+@Component({
+    tag: 'limel-example-text-editor-prefilled-value',
+    shadow: true,
+})
+export class BasicTextEditorExample {
+    @State()
+    private text: { html: string } = { html: '' };
+
+    //     @State()
+    //     private value = `
+    // 1. First ordered list item
+    // 2. Another item
+    //     * Unordered sub-list
+    // 1. Actual numbers don't matter, just that it's a number
+    //     1. Ordered sub-list
+    // 4. And another item.
+
+    //     You can have properly indented paragraphs within list items. Notice the blank line above, and the leading spaces (at least one, but we'll use three here to also align the raw Markdown).
+
+    //     To have a line break without a paragraph, you will need to use two trailing spaces.
+    //     Note that this line is separate, but within the same paragraph.
+    //     (This is contrary to the typical GFM line break behaviour, where trailing spaces are not required.)
+
+    // * Unordered list can use asterisks
+    // - Or minuses
+    // + Or pluses
+    // `;
+
```

Comment:
☝️ that should also mean that we probably need some sort of support for props like `disabled`, `readonly` (maybe)?

As well as support for `placeholder` and `label` perhaps?

Reply:
Yeah, true. I think we should focus on the basic functionality first though. We can add support for these later. Let's try to not make this first PR the mother of all PRs, OK? 🙏

Reply:
Solved in the other PR

Reply:
I feel using `helperText` might be an overkill. Also perhaps `label` is an overkill. I don't know. But it's so common in forms, so we maybe need to do something about it. Another thing we need is `invalid` but let's talk about them in your other PR.